src/lib/mongoBackend/MongoCommonSubscription.cpp

```diff
@@ -202,33 +202,43 @@ void setEntities(const Subscription& sub, BSONObjBuilder* b)
   BSONArrayBuilder entities;
   for (unsigned int ix = 0; ix < sub.subject.entities.size(); ++ix)
   {
-    EntID en = sub.subject.entities[ix];
+    EntID       en        = sub.subject.entities[ix];
+    std::string finalId;
+    std::string finalType;
+    std::string isIdPattern;
+    bool        isTypePattern = false;
```

Comment:
Very funny, i was caught for it and was the cause of the last tests failing ...
Actually it is in that way in the model description https://fiware-orion.readthedocs.io/en/develop/admin/database_model/index.html#csubs-collection
Maybe for another refactor PR ? (the impact could be big)